lib/api/core/models/repositories/pluginRepository.js

```diff
@@ -0,0 +1,94 @@
+const
+  _ = require('lodash'),
+  Repository = require('./repository'),
+  Data = require('../plugin/data');
+
+class PluginRepository extends Repository {
+  constructor (kuzzle, pluginName) {
+    super(kuzzle, pluginName);
+
+    // We set the collection to null on purpose to force plugins
+    // to specify the collection in each calls
+    this.collection = null;
+    this.ObjectConstructor = Data;
+  }
+
+  init (options) {
+    if (options instanceof Object && !Array.isArray(options) && options.cacheEngine !== null) {
```

Comment:
No, but we want to be sure it is a "real" object, that's all, why would it be deopted ?